src/main/java/frc/team2412/robot/subsystems/ArmSubsystem.java

```diff
@@ -1,5 +1,155 @@
 package frc.team2412.robot.subsystems;
 
+import static frc.team2412.robot.Hardware.*;
+import static frc.team2412.robot.subsystems.ArmSubsystem.ArmConstants.*;
+
+import com.revrobotics.CANSparkMax;
+import com.revrobotics.CANSparkMax.IdleMode;
+import com.revrobotics.CANSparkMaxLowLevel.MotorType;
+import edu.wpi.first.math.controller.ProfiledPIDController;
+import edu.wpi.first.math.trajectory.TrapezoidProfile.Constraints;
+import edu.wpi.first.wpilibj.Encoder;
 import edu.wpi.first.wpilibj2.command.SubsystemBase;
 
-public class ArmSubsystem extends SubsystemBase {}
+public class ArmSubsystem extends SubsystemBase {
+
+	// Constants
+
+	public static class ArmConstants { // To find values later
+		// Mech problem basically
+		public static final int ARM_LENGTH = 0;
+		public static final int VIRTUAL_BAR_LENGTH = 0;
+
+		public static final int K_P = 0;
+		public static final int K_I = 0;
+		public static final int K_D = 0;
+
+		// Trapezoid hee hee
+		public static final int ARM_POS_TOLERANCE = 0;
+		public static final int WRIST_POS_TOLERANCE = 0;
+
+		public static final int ARM_VELOCITY_TOLERANCE = 0;
+		public static final int WRIST_VELOCITY_TOLERANCE = 0;
+
+		public static final double MAX_ARM_VELOCITY = 5;
+		public static final double MAX_ARM_ACCELERATION = 5;
+
+		public static final double MAX_WRIST_VELOCITY = 5;
+		public static final double MAX_WRIST_ACCELERATION = 5;
+
+		public static final Constraints ARM_CONSTRAINTS =
+				new Constraints(MAX_ARM_ACCELERATION, MAX_ARM_VELOCITY);
+		public static final Constraints WRIST_CONSTRAINTS =
+				new Constraints(MAX_WRIST_ACCELERATION, MAX_WRIST_VELOCITY);
+
+		// Arm Positions
+
+		public static final double HIGH_NODE_ARM_ANGLE = 56.99;
+		public static final double MIDDLE_NODE_ARM_ANGLE = 92;
+		public static final double LOW_GRAB_ARM_ANGLE = 0;
+		public static final double RETRACT_ARM_ANGLE = 169;
+		public static final double SUBSTATION_ARM_ANGLE = 80;
+
+		// find values later
+		public static final double HIGH_NODE_WRIST_ANGLE = 0;
+		public static final double MIDDLE_NODE_WRIST_ANGLE = 0;
+		public static final double LOW_GRAB_WRIST_ANGLE = 0;
+		public static final double RETRACT_WRIST_ANGLE = 90;
+		public static final double SUBSTATION_WRIST_ANGLE = 0;
+
+		// LIKELY DONT NEED
+		// Apparently, we wanted the node position to be in ticks.
+		// We need an inches to ticks converter.
```

Comment:
For all these items that still need to be tackled, add "TODO:" in the comment.  That makes them harder to miss.